sys/androidmedia/gstamc.c

```diff
@@ -1238,7 +1238,7 @@ scan_codecs (GstPlugin * plugin)
     }
 
     if (g_str_has_prefix (name_str, "OMX.ARICENT.")
-        || g_str_has_prefix (name_str, "OMX.MTK.AUDIO")) {
+        || g_str_has_prefix (name_str, "OMX.MTK.AUDIO.DECODER.DSPAAC")) {
```

Comment:
I suppose that it did solve, this block can be removed, it's just it wasn't tested very well, and also for aac there're google's decoders. I can remove this OR at all, maybe it's time. But it will imply much more cases.

Reply:
I that the sync mode introduced here is expected to fix also the mtk aac decoder issue. I think we should confirm it to avoid leaving code that is misleading or not necessary.